staging/src/k8s.io/api/autoscaling/v1/types.go

```diff
@@ -344,6 +356,14 @@ type ObjectMetricStatus struct {
 	MetricName string `json:"metricName" protobuf:"bytes,2,name=metricName"`
 	// currentValue is the current value of the metric (as a quantity).
 	CurrentValue resource.Quantity `json:"currentValue" protobuf:"bytes,3,name=currentValue"`
+
+	// selector is the string-encoded form of a standard kubernetes label selector for the given metric
```

Comment:
Explain what selector means to the user in this context.  Why would they set it, what happens if it is empty, etc.